### DISCUSSION TITLE
Student Developer Pack Ticket Support

### DISCUSSION THREAD

**Discussion by hobi-design** [community/community#110598] [GitHub Education]
### Select Topic Area

General

### Body

Hello, I applied for the Student Developer Pack nearly two weeks ago (20th February) via a support ticket containing my documents since the university I am attending hasn't provided us with official student email IDs.

Could anyone look into it and let me know how much longer the process might take?

Ticket ID: 2608568

Thank you.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @OutOfTokens , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by hobi-design** [reply]
Thank you and happy to be here. :)

Is there anything else I can do to get things going?

**Comment by hobi-design** [comment]
Update: I tried to apply for the student pack twice, first using my college-issued ID card and then with my college student registration form.

Both times it was rejected. My college has not issued a student email ID and I created a ticket for this more than three weeks ago but never got a response to it. All students use their personal GMAIL email address for all college work.

The registration form I submitted has the gmail address that is attached to my Github account. 

Not sure what else to do.

**Comment by moozunch** [comment]
tried to help answer your reply about getting rejected, do you have any rejected messages of why those things happen? here are example of mine 
![image](https://github.com/community/community/assets/112236945/5a265b2a-53f9-455d-93b6-2799a394d6c1)
after I follow all the instructions, even tho my id card is blurred (kind of, pretty sure the pics doesn't really clear) but finally I got accepted to the global campus and get the benefit, tried to check yours.

**Reply by hobi-design** [reply]
Hi, thank you for getting back to me. The first bullet point in the list of reasons is **"Attendees of this school must verify and use their academic email address."**

But as I have stated in my support ticket, our college hasn't provided us with official emails yet (rarely does from what I've heard). I am a first year student.

**Comment by hobi-design** [comment]
Am I ever going to get any help with this? :/
